### PR TITLE
Merge ipmi-console/racadmsim

### PR DESCRIPTION
**Pull request by turtle-fly**
1. Keep ipmi-console and racadmsim command line the same
2. Rename python package infrasim.ipmicons to infrasim.ipmiconsole,
create python pcakage infrasim.racadmsim
3. Move original ipmiconsole.py to infrasim/ipmiconsole/__init__.py, move
original racadmsim.py to infrasim/racadmsim/__init__.py

### CODE REVIEW

**Comment by turtle-fly** [comment]
Do NOT merge now. There will be code refine on racadmsim as well.

**Comment by XiaowenJiang** [comment]
cool

**Comment by turtle-fly** [comment]
@fub2 @xiar @XiaowenJiang @chenge3 @WinnieXU1 @JuneZhou19 please help review, this is to refactor infrasim package constructure, by making functioning module a respective package.

File: infrasim/ipmiconsole/__init__.py [hunk 1]
```diff
@@ -9,19 +9,17 @@
 import threading
```
Comment (fub2) [inline infrasim/ipmiconsole/__init__.py]: making ipmiconsole a lib is good idea!

**Comment by fub2** [comment]
Everything looks good! @xiar any potential impact to dell server simulation you're working on?

**Pull request merged**